### ISSUE TITLE
Accessibility - Dataverse - Level One Heading

### ISSUE THREAD

**Issue by tander08** [QualitativeDataRepository/dataverse#91]
### Homepage: https://data.qdr.syr.edu/
- [x] Whole Page: 
![image](https://user-images.githubusercontent.com/89411909/134694293-bb54a763-3616-4af3-8eb8-ede6818c88d3.png)

### Advanced Search: https://data.qdr.syr.edu/dataverse/main/search
- [x] Same as Above

### (Sample Page): Data for: Qualitative Data Sharing: Participant Understanding, Motivation, and Consent: https://data.qdr.syr.edu/dataset.xhtml?persistentId=doi:10.5064/F6YYA3O3
- [x] Same as Above

**Comment by qqmyers** [comment]
I made the collection name shown at the top of these pages into an H1. I don't think this is a general solution as that header is different for IQSS/has the main menu in it, etc. but I think it works for QDR. (Not sure if this is in another issue, but the top area needs a landmark and I chose role="banner" for QDR since our navigation menu is separate. That does mean that the H1 for the page is in the "banner' section right now. Not sure if that's acceptable - if not we may need to reshuffle some of the outer divs on the page to put this info in the main content section, etc. That said, it might be worth waiting on that to see what general solution will work for IQSS (again since they have nav elements in this section, etc.)).
